src/components/StudentsTable/StudentsTable.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+
+
+const StudentsTable = ({rows, columns}) => (
+    <div style={{ height: 620, width: '100%' }}>
+      <DataGrid rows={rows} columns={columns} pageSize={10} checkboxSelection/>
```

Comment:
Checkbox думаю пока лишний, т.к. пока для него нету функционала
```suggestion
      <DataGrid rows={rows} columns={columns} pageSize={10}/>
```